tests/test_indexer.py

```diff
@@ -417,6 +445,10 @@ def setUpClass(cls):
     def process_new_indexable_object(self, event, logger):
         process_new_gs_indexable_object(event, logger)
 
+    def test_process_removed_object(self):
```

Comment:
Maybe `@self.skip("de-index for s3 only")`. Also add a todo to add delete event for GS.